### PR TITLE
Upgrade to 13.9.3

### PR DESCRIPTION
**Pull request by sachilles**
GitLab Patch Release: 13.9.3 (see <https://about.gitlab.com/releases/2021/03/08/gitlab-13-9-3-released/>)

@solidnerd @sameersbn There is now a great deal to do here. Please merge the pull requests!

### CODE REVIEW

File: docs/docker-compose-keycloak.yml [hunk 1]
```diff
@@ -22,7 +22,7 @@ services:
 
   gitlab:
     restart: always
-    image: sameersbn/gitlab:13.8.2
+    image: sameersbn/gitlab:13.9.0
```
Comment (MRigal) [inline docs/docker-compose-keycloak.yml:25]: missing the minor update here...

Comment (sachilles) [reply docs/docker-compose-keycloak.yml:25]: Sorry. I must have missed this file. I promise to do better next time.

Comment (sachilles) [reply docs/docker-compose-keycloak.yml:25]: The files `contrib/docker-swarm/docker-compose.yml` and `docker-compose.swarm.yml` are also affected.

Comment (sameersbn) [reply docs/docker-compose-keycloak.yml:25]: fixed in https://github.com/sameersbn/docker-gitlab/pull/2319

**Pull request merged**